utils/pip-allow-failures.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+PIP_COMMAND=${PIP_COMMAND:-pip}
+PIP_OPTIONS=${PIP_OPTIONS:-}
+REQUIREMENTS_FILE=$1
+
+while read dependency; do
+    dependency_stripped="$(echo "${dependency}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+    # Skip comments
+    if [[ $dependency_stripped == \#* ]]; then
+        continue
+    # Skip blank lines
+    elif [ -z "$dependency_stripped" ]; then
+        continue
+    else
+        if $PIP_COMMAND install $PIP_OPTIONS "$dependency_stripped"; then
```

Comment:
Since you are already displaying the result afterwards, maybe you can redirect the output to `/dev/null` ?